WorldOfPAYBACK/Model/FilterOption.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  FilterOption.swift
+//  WorldOfPAYBACK
+//
+//  Created by Kacper Rubaszewski on 22/02/2023.
+//
+
+import Foundation
+
+enum FilterOption:String{
+    
+    static var allFilters: [FilterOption] {
+        return [.All, .ONE, .TWO, .THREE]
+    }
+    case All = "All"
```

Comment:
hint: more common would be `all`, `one` etc. 

Check this https://github.com/kodecocodes/swift-style-guide at least many style guides I had encountered were very similar :)